roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
+- to be well organised and committed
+- to be accountable for progress and showcases back to the business
```

Comment:
I think we historically have had expectations up front on how things should be run, we also have learnt alot with BizOps on how to make things transparent. We currently have Academy as a BizOps goal, we should just solidify this and maintain visibility by keeping it as such.